### PR TITLE
Fix Document/Elements/Paste-Style: Fix - When paste styling with globals (PT#12701)

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: assets/dev/js/editor/document/elements/commands/paste-style.js [hunk 1]
```diff
@@ -96,26 +96,55 @@ export class PasteStyle extends CommandHistory {
...
+			$e.run( 'document/globals/settings', {
+				container: targetContainer,
+				settings: globals,
+				options: {
+					external: true,
+					render: false,
+				},
+			} );
+
+			targetContainer.panel.refresh();
```
Comment (iNewLegend) [inline assets/dev/js/editor/document/elements/commands/paste-style.js:139]: @matipojo

File: assets/dev/js/editor/document/elements/commands/paste-style.js [hunk 2]
```diff
@@ -96,26 +96,55 @@ export class PasteStyle extends CommandHistory {
...
+	}
 
-				// BC: Deprecated since 2.8.0 - use `$e.hooks`.
-				elementor.channels.data.trigger( 'element:before:paste:style', targetContainer.model );
+	/**
+	 * @param {Container} targetContainer
+	 * @param {{}} settings
+	 */
+	pasteStyle( targetContainer, settings ) {
+		targetContainer.view.allowRender = false;
```
Comment (matipojo) [inline assets/dev/js/editor/document/elements/commands/paste-style.js:109]: Remove, the settings has `render: false`.

File: assets/dev/js/editor/document/elements/commands/paste-style.js [hunk 3]
```diff
@@ -96,26 +96,55 @@ export class PasteStyle extends CommandHistory {
...
+				},
+			} );
+
+			targetContainer.panel.refresh();
+		}
+
+		// BC: Deprecated since 2.8.0 - use `$e.hooks`.
+		elementor.channels.data.trigger( 'element:after:paste:style', targetContainer.model );
+
+		targetContainer.view.allowRender = true;
```
Comment (matipojo) [inline assets/dev/js/editor/document/elements/commands/paste-style.js:145]: Remove, the settings has `render: false`.

**Review by matipojo — CHANGES_REQUESTED** [review]
Please add tests.

File: assets/dev/js/editor/document/elements/commands/paste-style.js (hunk 3 above)
Comment (iNewLegend) [reply assets/dev/js/editor/document/elements/commands/paste-style.js]: Handled.

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**